### DISCUSSION TITLE
Thanks for a great QR Code library

### DISCUSSION THREAD

**Discussion by onmyway133** [dagronf/QRCode#57] [General]
Hi, hope are you doing well. I just want to say that you're doing great work with this library. We use this in our completely free app https://indiegoodies.com/prettyqr to let user craft beautiful QR code. 

Keep up the good work 🚀

**Comment by dagronf** [comment]
@onmyway133 Thank you so much!  It's always great to see it being used in wonderful apps.

**Reply by onmyway133** [reply]
@dagronf You should put Sponsorship to this project, I will be happy to donate
